test/alternative_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative "spec_helper.rb"
+require "update-alternatives/model/alternative"
+
+describe UpdateAlternatives::Alternative do
+
+  describe ".all" do
+    subject { UpdateAlternatives::Alternative.all }
+
+    it "returns an array of Alternative objects" do
+      alternatives_pip_with_two_choices_stub
+      expect(subject.class).to eq Array
+      subject.all? { |e| expect(e).to be_a(UpdateAlternatives::Alternative) }
+    end
+
+    context "alternative with one choice" do
```

Comment:
Here you are not actually testing `.all`. This actually looks like a test for `.load`. Same applies to next context.